src/Sql/dbo/Tables/Organization.sql

```diff
@@ -42,6 +42,7 @@
     [MaxAutoscaleSeats]             INT              NULL,
     [UseKeyConnector]               BIT              NOT NULL,
     [UseScim]                       BIT              NOT NULL CONSTRAINT [DF_Organization_UseScim] DEFAULT (0),
+    [UseSecretsManager]             BIT              NOT NULL CONSTRAINT [DF_Organization_UseSecretsManager] DEFAULT (0),
```

Comment:
Not sure why we name this constraint, but I followed the UseScim pattern.

Reply:
I believe this is a default constraint for specifying the default value for the column. 

If the name isn't specified, an auto generated name for the constraints gets generated. 

When backing out the change, it's easier to specify deleting the pre named constraint before removing the column instead of trying to fetch the auto generated name for deletion.